File: mvp.tickets.domain/Enums/Permissions.cs
namespace mvp.tickets.domain.Enums
{
    [Flags]
    public enum Permissions
    {
        None     = 0b000000000,
        User     = 0b000000001,
        Admin    = 0b000000010,
        Employee = 0b000000100
    }
}

File: mvp.tickets.web/ClientApp/src/Enums/Permissions.ts
export enum Permissions
{
    None     = 0,
    User     = 1 << 0,
    Admin    = 1 << 1,
    Employee = 1 << 2,
}

export function hasPermission(source: Permissions, permission: Permissions) : boolean {
    return permission === (source & permission);
}